deploy/olm-catalog/kogito-operator/manifests/app.kiegroup.org_kogitotrusty_crd.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
What is the difference against: "org_kogitotrusty_crd.yaml "? Why do we need two CRDs here?